packages/core/src/services/sessionSummaryHelper.ts

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
optional nit : having a file name ending in helper is a little non-standard. is there a reason this isn't in ChatRecordingService. Can still be a top level method if you want. If you want to keep it its own file would rename to sessionSummaryUtils.ts

Reply:
change to sessionSummaryUtils.ts, didn't want to deal with circular deps